src/views/NewArticle.js

```diff
@@ -0,0 +1,195 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import RichTextEditor from "react-rte";
+import Input from "../components/common/Inputs/TextInput";
+import Button from "../components/common/Buttons/confirm";
+import Tag from "../components/common/Tags/NewTag";
+import { toolbarConfig } from "../components/common/editor/editorConfig";
+import {
+  handleInputField,
+  newArticle,
+  handleCreateTag,
+  removeTag
+} from "../redux/actions/newArticle";
+
+export const mapDispatchToProps = dispatch => ({
+  onInputChange: field => dispatch(handleInputField(field)),
+  newTag: tag => dispatch(handleCreateTag(tag)),
+  createArticle: article => dispatch(newArticle(article)),
+  handleRemoveTag: tag => dispatch(removeTag(tag))
+});
+
+const mapStateToProps = state => ({
+  response: state.article.response,
+  article: state.article
+});
+
+export class CreateArticle extends Component {
+  constructor() {
+    super();
+    this.state = {
+      tag: "",
+      error: "",
+      value: RichTextEditor.createEmptyValue()
+    };
+    this.submitArticle = this.submitArticle.bind(this);
```

Comment:
User arrow functions and you will not need to bind all these properties to ```this``` context